exporters/otlp/internal/otlpconfig/options.go

```diff
@@ -42,31 +42,16 @@ const (
 	// DefaultTimeout is a default max waiting time for the backend to process
 	// each span or metrics batch.
 	DefaultTimeout time.Duration = 10 * time.Second
-	// DefaultServiceConfig is the gRPC service config used if none is
-	// provided by the user.
-	DefaultServiceConfig = `{
-	"methodConfig":[{
-		"name":[
-			{ "service":"opentelemetry.proto.collector.metrics.v1.MetricsService" },
-			{ "service":"opentelemetry.proto.collector.trace.v1.TraceService" }
-		],
-		"retryPolicy":{
-			"MaxAttempts":5,
-			"InitialBackoff":"0.3s",
-			"MaxBackoff":"5s",
-			"BackoffMultiplier":2,
-			"RetryableStatusCodes":[
-				"CANCELLED",
-				"DEADLINE_EXCEEDED",
-				"RESOURCE_EXHAUSTED",
-				"ABORTED",
-				"OUT_OF_RANGE",
-				"UNAVAILABLE",
-				"DATA_LOSS"
-			]
-		}
-	}]
-}`
+)
+
+var (
+	// DefaultRetrySettings is a default settings for the retry policy.
+	DefaultRetrySettings = otlp.RetrySettings{
```

Comment:
This is a global variable. I think it should not.

1. After looking at the codebase I have a feeling that all default is immutable and constants (which I like very much)
2. It can cause data races - if setting and accessing `DefaultRetrySettings` would be done in different goroutines

Reply:
Can we unexport this?

Reply:
This resides in an `Internal` package, It is our responsibility to treat _default_ `var` as immutable. 

If that is still not enough, I can just transform this in several `const` and build this struct when needed.

Reply:
> This resides in an `Internal` package, It is our responsibility to treat _default_ `var` as immutable.
> 
> If that is still not enough, I can just transform this in several `const` and build this struct when needed.

I missed this was in `internal`. SGTM leaving as is.

Reply:
For me it can leave as it as well.
However, I think that unexporting it would be also good as it is used only in this package anyway.
I am always in favor of reducing the API surface.

Reply:
I've unexported it. Don't know why I thought this was used by other packages.